sdk/python/feast/config.py

```diff
@@ -29,6 +29,7 @@
 from feast.constants import ConfigOptions as opt
 
 _logger = logging.getLogger(__name__)
+_UNSET = object()
```

Comment:
How is the `object()` type used here? Is there any reason why we can't use `default=None`?

Reply:
Is a signleton, just like `configparser._UNSET`. If you use `None`, either you can't use `None` as default value, or you'll get `None` even if you don't want default value and do expect an exception